spec/system/verification_spec.rb

```diff
@@ -8,6 +8,103 @@
 
   before { AidApplicationSearch.refresh }
 
+  context 'when an assister is logged in' do
+    context 'when changes are made' do
+      it 'redirects to the dashboard' do
+        sign_in assister
+        visit root_path
+
+        within '.searchbar' do
+          fill_in "q", with: aid_application.application_number
+          click_on 'Search'
+        end
+
+        click_on aid_application.application_number
+
+        within '#application-navigation' do
+          click_on 'Verify'
+        end
+
+        fill_in "Full name", with: "New Name"
+        fill_in "aid_application[birthday(1i)]", with: "2000"
+        fill_in "aid_application[birthday(2i)]", with: "2"
+        fill_in "aid_application[birthday(3i)]", with: "2"
+        select "Bisexual", from: "Sexual orientation"
+        select "Another gender identity", from: "Gender"
+
+        click_on 'Save and exit'
+
+        expect(page).to have_content 'Start a new application'
+
+        aid_application = AidApplication.last
+        expect(aid_application).to have_attributes(
+                                     name: "New Name",
+                                     birthday: "02-02-2000".to_date,
+                                     sexual_orientation: "Bisexual",
+                                     gender: "Another gender identity",
+                                     )
+
+      end
+    end
+
+    context 'when the application is a duplicate of an approved application' do
+      before do
+        duplicate = aid_application.dup
+        duplicate.update!(
+          application_number: duplicate.generate_application_number,
+          approved_at: 1.day.ago,
+          approver: supervisor
+        )
+      end
+
+      it 'shows the duplicate application screen' do
+        sign_in assister
+        visit root_path
+
+        within '.searchbar' do
+          fill_in "q", with: aid_application.application_number
+          click_on 'Search'
+        end
+
+        click_on aid_application.application_number
+
+        within '#application-navigation' do
+          click_on 'Verify'
+        end
+
+        click_on 'Save and exit'
+
+        expect(page).to have_content 'This applicant has been identified as a duplicate.'
+
+        click_on "Go back"
+
+        expect(current_path).to eq edit_organization_aid_application_verification_path(aid_application.organization, aid_application, locale: 'en')
+      end
+    end
```

Comment:
You probably don't need this test because that behavior isn't too closely tied into the fact that a non-supervisor also cannot access this page. 

It's a toss-up, but I suggest it because System Tests are the slowest kinds of tests, so while they are very powerful, they should be used sparingly. For purely a question of access control, it's acceptable to do that as part of a Controller test.

Reply:
I just kept the test for assisters not being able to unpause in the system test. Does that work?

Reply:
👍